java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_2/BirthDayValidator.java

```diff
@@ -0,0 +1,24 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class BirthDayValidator implements ValidatorInterface {
+    private final int maxDay = 31;
+    private final int minDay = 1;
+
+    private int birthDay;
+
+    public BirthDayValidator(int birthDay) {
+        this.birthDay = birthDay;
+    }
+
+
+    @Override
+    public boolean isValid() {
```

Comment:
All validators need to have a method isValid that accepts **`a user account object`**